Makefile

```diff
@@ -289,4 +289,11 @@ logit-filters:
 		-v $(CURDIR)/config/logit/output:/output:rw \
 		-w /mnt \
 		jruby:9.1-alpine ./scripts/generate_logit_filters.sh $(LOGSEARCH_BOSHRELEASE_TAG) $(LOGSEARCH_FOR_CLOUDFOUNDRY_TAG)
+	@echo "Making changes to $(CURDIR)/config/logit/output/generated_logit_filters.conf"
```

Comment:
Can we do this in the `./scripts/generate_logit_filters.sh` file instead?

Reply:
Yep, good call. Resolved in the amended commit.